scribble-lib/scribble/private/lang-params.rkt

```diff
@@ -0,0 +1,12 @@
+#lang racket/base
+
+(provide (all-defined-out))
+
+;; Some latex formats have different requirements on how
+;;   figures, citations, etc. are displayed. This allows different
+;;   scribble langs to handle them.
+
+(define default-label-text (make-parameter "Figure"))
+(define default-label-sep (make-parameter ": "))
+(define default-caption-style (make-parameter #f))
```

Comment:
Can you rename these first 3 to `default-figure-....`

Reply:
Also, add a comment describing the type of each parameter

Reply:
Agreed, done.